### DISCUSSION TITLE
Any chance to have Stapler installable with Homebrew?

### DISCUSSION THREAD

**Discussion by dlebouc** [gingerbeardman/stapler#21] [Q&A] [answered]
The main interest is update is easier, simple and can be regular.
Thanks in advance

**Comment by gingerbeardman** [comment, marked as answer]
Once things settle down (new icon and various fixes) I'll try to look into it.

The problem is it's an extra task that needs to be done with every release, and it's not very enjoyable. So, unless I can automate it... I will probably leave it to somebody else.

I do not need to be involved, anybody else is free to do it and take ownership of that formula.
